var/spack/repos/builtin/packages/cppad/package.py

```diff
@@ -9,20 +9,38 @@
 class Cppad(CMakePackage):
     """A Package for Differentiation of C++ Algorithms."""
 
-    homepage = "https://www.coin-or.org/CppAD/"
-    url = "http://www.coin-or.org/download/source/CppAD/cppad-20170114.gpl.tgz"
+    maintainers("whart222")
+
+    homepage = "https://github.com/coin-or/CppAD"
+    url = "https://github.com/coin-or/CppAD/archive/refs/tags/20240000.4.tar.gz"
     git = "https://github.com/coin-or/CppAD.git"
 
```

Comment:
```suggestion
    homepage = "https://github.com/coin-or/CppAD"
    url = "https://github.com/coin-or/CppAD/archive/refs/tags/20240000.4.tar.gz"
    git = "https://github.com/coin-or/CppAD.git"
    
    maintainers("whart222")

```

Following the common format, the `maintainers()` directive should go under homepage, url, and git variables and before the first `version()` directive.

Reply:
Alec is correct in that the convention is to have directives appear **after** properties.

Reply:
Done.